app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
```

Comment:
This name suggests that we are using this class for not only the account deletion feature but also for other features. Is it right?

Reply:
I will reply in https://github.com/3scale/porta/pull/574#discussion_r254211723 😄